io_capture.py

```diff
@@ -0,0 +1,148 @@
+"""
+PEP-8 Conforming program to capture calls' IO across functions/class methods/inner classes in all
+    the files in the 'example_projects' DIR.
+"""
+
+import glob
+import os
+import importlib
+import inspect
```

Comment:
Can you restructure the files:
- `example_project`
- `src` for driver and capture code
- add a `requirement.txt` and list all dependencies?